### DISCUSSION TITLE
Havic DH "The Hunt" never appearing

### DISCUSSION THREAD

**Discussion by rhaimi** [Hekili/hekili#2101] [Q&A]
Hello,

Thanks for the very nice addon.
I have a question about "The Hunt" for demon hunter Havoc.
It is our top priority spell, but it is never showing in Hekili.
Did I miss something in the setting for it never appearing ?
I use default config

Regards

**Comment by Hekili** [comment]
You can submit an issue ticket regarding Recommendations and provide the required snapshot per those ticket instructions.

**Comment by Zeuslight56** [comment]
Do you have cooldowns enabled?  Because I have no problem.

**Comment by brandodrake** [comment]
I've noticed that as well but haven't found an answer. Wanted to ask if you know in the rotation, one of the first buttons is a green vial looking ability but I cannot find it anywhere in my spell book. Do you know what that is?

**Reply by Hekili** [reply]
This will almost always be a trinket or usable piece of gear.  Pausing (`ALT-SHIFT-P`, by default) lets you mouse over the icon to see what it is and why it's showing at any time.  Tap the keybind again to unpause.

**Reply by brandodrake** [reply]
Thanks!
